app/.eslintrc.js

```diff
@@ -0,0 +1,320 @@
+        "no-octal-escape": "error",
+        "no-self-compare": "error",
+        "no-unmodified-loop-condition": "error",
+        "no-useless-concat": "error",
+        "no-useless-return": "error",
+        "prefer-promise-reject-errors": "error",
+        "radix": "error",
+        "yoda": "error",
+
+        // eslint: Variables
+        "no-shadow": "error",
+
+        // eslint: Stylistic Issues
+        "array-bracket-newline": [
+            "error",
+            "consistent"
+        ],
+        "array-bracket-spacing": "error",
+        "comma-dangle": [
+            "error",
+            "always-multiline"
+        ],
+        "comma-style": "error",
+        "computed-property-spacing": "error",
+        "eol-last": "error",
+        "function-call-argument-newline": [
+            "error",
+            "consistent"
+        ],
+        "key-spacing": "error",
+        "new-parens": "error",
+        "no-bitwise": "error",
+        "no-multi-assign": "error",
+        "no-multiple-empty-lines": [
+            "error",
+            {
+                "max": 1,
+                "maxEOF": 0
+            }
+        ],
+        "no-trailing-spaces": "error",
+        "no-unneeded-ternary": "error",
+        "no-whitespace-before-property": "error",
+        "object-curly-newline": [
+            "error",
+            {
+                "consistent": true
+            }
+        ],
+        "object-curly-spacing": [
+            "error",
+            "never"
+        ],
+        "object-property-newline": [
+            "error",
+            {
+                "allowAllPropertiesOnSameLine": true
+            }
+        ],
+        "one-var-declaration-per-line": [
+            "error",
+            "always"
+        ],
+        "operator-assignment": "error",
+        "padding-line-between-statements": [
+            "error",
+            {"blankLine": "always", "prev": "import", "next": "*"},
+            {"blankLine": "never", "prev": "import", "next": "import"},
+            {"blankLine": "always", "prev": "multiline-block-like", "next": "*"},
+            {"blankLine": "always", "prev": "*", "next": "multiline-block-like"},
+        ],
+        "prefer-object-spread": "error",
+        "quote-props": [
+            "error",
+            "as-needed"
+        ],
+        "semi-spacing": "error",
+        "semi-style": "error",
+        "space-in-parens": "error",
+        "space-infix-ops": "error",
+        "space-unary-ops": "error",
+        "spaced-comment": "error",
+        "switch-colon-spacing": "error",
+
+        // eslint: ECMAScript 6
+        "arrow-parens": "error",
+        "arrow-spacing": "error",
+        "no-confusing-arrow": "error",
+        "no-duplicate-imports": "error",
+        "no-useless-rename": "error",
+        "object-shorthand": "error",
+        "prefer-const": "error",
+        "prefer-rest-params": "error",
+        "prefer-spread": "error",
+        "prefer-template": "error",
```

Comment:
Will need this for some string concatination, bit overkill to use templates for everything. Suggest turning to warning.

Reply:
Templates stop feeling like overkill very soon once you start using them as the standard way. But I have nothing against switching it to warning.

Reply:
Yeah lets go ahead with warning for now, if they're proving bothersome we can make it error.